addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -124,6 +124,7 @@ function factory(dependencies) {
                     manager: [['link', this]],
                     thread: [['link', thread]],
                 });
+                await thread.fetchData();
```

Comment:
Although `fetchData` is in `thread`, this makes only sense for chatter threads. I would only do this fetch on `thread.model !== mail.channel`